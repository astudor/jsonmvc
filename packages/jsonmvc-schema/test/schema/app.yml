foo:
  $ref: types/meta.yml
bar:
  $ref: nested/collection/bar.yml
baz:
  $ref: nested/collection/baz.json
boo:
  $ref: nested/collection/boo.yml
users:
  type: object
  patternProperties:
    "^[A-Za-z0-9]+$":
      type: object
      properties:
        name:
          type: string
collection:
  allOf:
  - $ref: types/meta.yml
  - $ref: nested/collection/bar.yml
  - type: object
    properties:
      bam:
        type: integer

other:
  type: integer
  default: __SAMPLE_PLACEHOLDER__