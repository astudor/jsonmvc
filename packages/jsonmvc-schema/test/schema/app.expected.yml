---
foo:
  type: object
  properties:
    foo:
      type: object
      properties:
        bar:
          type: string
          value: 123
bar:
  type: integer
baz:
  baz:
    type: string
boo:
  boo:
    type: integer
users:
  type: object
  patternProperties:
    "^[A-Za-z0-9]+$":
      type: object
      properties:
        name:
          type: string
collection:
  type: object
  properties:
    foo:
      type: object
      properties:
        bar:
          type: string
          value: 123
    bam:
      type: integer
    baz:
      type: number
other:
  type: integer
  default: 123